docs/build_commands.rst

```diff
@@ -368,6 +368,19 @@ The following ``pip`` package specification formats are supported:
 * The ``git+`` and ``hg+`` links (the git and mercurial are installed as build
   dependency automatically)
 
+  .. note::
+     In Ubuntu you may also need to install ``ca-certificates`` package
+     to be able to install ``git+/hg+`` python packages:
```

Comment:
Yes. We install git and hg automatically. Should probably add `ca-certificates` to the list (#58)